### ISSUE TITLE
TCGA DataLoader initial_work() function looks for expression files which are not included

### ISSUE THREAD

**Issue by ibeyreli** [iron-lion/tmp_model#2]
Hello, Really nice work with the model development. 

I wished to use the TCGA pretrained model for my own dataset. I have written my own 5-fold 5-repeats cross validation in addition similar to your scheme and was succesful to get some results. Different from your approach, I enforced the model traverse my whole test set to get a prediction for each sample I have. During this process I read really good episode performances, around 0.85 which is expected. But when I get median of predictions (note that this is a binary prediction task and I use the predicted classes not prediction scores) to get a final list, the performance drops to ~0.75. I checked the predicted class ratio as my data is imbalanced but the model seems to work there as it does not predict only one class.

As this is unexpected, I wanted to replicate your results with my own cross validation scheme as well. I have downloaded the `data` folder in the repository and tried to run the initial_work() function to get .H5 files. Unfortunately no success. In tmp_loader.py line 93, the code looks for 'exp' in the file names, but when I check the data folder manually those files are not included. 

1. Have you also observed such a drop in accuracy, when you pushed all of your data in different episodes and combine the results? What might be the reason of such a drop in your opinion?
2. To be able to replicate the results and making sure that my cross validation also works as it should, could you please check if the data folder is, indeed, whole and contains the all needed files? Otherwise, if I have to find them in another place, could you kindly point me to that direction? 

Thanks in advance,

**Comment by iron-lion** [comment]
Hi @ibeyreli , thank you for the issue.

First of all, on this commit, I updated the missing script for TCGA data setup.
I tested it on a Linux environment.
[#af68ed9 ](https://github.com/iron-lion/tmp_model/commit/af68ed937eb3f26fa919e5869a9a4e6686baa2b1)

TCGA data directory must follow this structure.

TCGA/
:  tcgablca/
:  :  metadata.cart.2021-05-19.json
:  :  sample.tsv  
:  :  exp/[BARCODE]/[BARCODE].FPKM.txt



For the accuracy drop, we still need more discussion.
